### ISSUE TITLE
Skript not starting with Paper

### ISSUE THREAD

**Issue by Remixful** [SkriptLang/Skript#2032]
Minecraft 1.13.2
Paper-1.13.2-b624
Skript 2.3.6

Here is the error log:
https://pastebin.com/raw/FfYyECrm

**Comment by Remixful** [comment]
Doesn't seem to work with b624 either. It's recommended to use Paper but I don't even know which version to use.. there's so many.

**Comment by ShaneBeee** [comment]
I can't reproduce this
```
> ver skript
[19:39:16 INFO]: Skript version 2.3.6
[19:39:16 INFO]: Customize Minecraft's mechanics with simple scripts written in plain English sentences.
[19:39:16 INFO]: Website: https://skriptlang.github.io/Skript
[19:39:16 INFO]: Authors: Njol, Mirreski, bensku, TheBentoBox, Snow-Pyon, Pikachu920, Nicofisi, tim740, Tuke-Nuke, TheLimeGlass, xXAndrew28Xx, Sashie, RoyCurtis, nfell2009, Syst3ms and Blueyescat
> ver
[19:39:17 INFO]: This server is running Paper version git-Paper-624 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)
[19:39:17 INFO]: Previous version: git-Paper-622 (MC: 1.13.2)
[19:39:17 INFO]: Checking version, please wait...
[19:39:18 INFO]: You are running the latest version
> 
```

I recommend stopping your server. Deleting plugins/Skript/materials.json then starting your server up.

**Comment by Remixful** [comment]
Not getting this error anymore after removing some files, can't recall how to reproduce.
